src/cmd_argument.rs

```diff
@@ -21,12 +22,30 @@ where
     }
 }
 
+/// Arguments of type [`OsString`] are passed to the child process
+/// as arguments.
```

Comment:
Maybe a doctest would be good here? Maybe something with the example from the PR description, i.e. using environment variables...